routing/integrated_routing_context_test.go

```diff
@@ -167,16 +193,27 @@ func (c *integratedRoutingContext) testPayment() ([]htlcAttempt, error) {
 			success: success,
 		})
 
+		fmt.Printf("Attempt %v: %v\n", success, route)
```

Comment:
note: remove when done debugging

Reply:
debug prints removed. same info is more or less logged by lnd anyway